relay-general/src/pii/utils.rs

```diff
@@ -0,0 +1,45 @@
+use lazycell::AtomicLazyCell;
+
+#[derive(Debug, Clone)]
+pub struct SerdeFriendlyLazyCell<T>(AtomicLazyCell<T>);
+
+impl<T> PartialEq for SerdeFriendlyLazyCell<T> {
+    fn eq(&self, _other: &Self) -> bool {
+        // for its uses in `PiiConfig` and `DataScrubbingConfig` it doesn't matter what is inside
```

Comment:
please review if this makes sense. the alternative I see is to manually impl PartialEq on datascrubbingconfig and piiconfig